bin/plugin/commands/common.js

```diff
@@ -41,44 +41,7 @@ exports.getModuleData = async ( modulesDir ) => {
 
 	return moduleFiles
 		.map( ( moduleFile ) => {
-			// Populate slug and focus based on file path.
-			const moduleDir = path.dirname( moduleFile );
-			const moduleData = {
-				slug: path.basename( moduleDir ),
-				focus: path.basename( path.dirname( moduleDir ) ),
-			};
-
-			// Map of module header => object property.
-			const headers = {
-				'Module Name': 'name',
-				Description: 'description',
-				Experimental: 'experimental',
-			};
-
-			// Populate name, description and experimental based on module file headers.
-			const fileContent = fs.readFileSync( moduleFile, 'utf8' );
-			const regex = new RegExp(
-				`^(?:[ \t]*<?php)?[ \t/*#@]*(${ Object.keys( headers ).join(
-					'|'
-				) }):(.*)$`,
-				'gmi'
-			);
-			let match = regex.exec( fileContent );
-			while ( match ) {
-				const content = match[ 2 ].trim();
-				const prop = headers[ match[ 1 ] ];
-				if ( content && prop ) {
-					moduleData[ prop ] = content;
-				}
-				match = regex.exec( fileContent );
-			}
-
-			// Parse experimental field into a boolean.
-			if ( typeof moduleData.experimental === 'string' ) {
-				moduleData.experimental =
-					moduleData.experimental.toLowerCase() === 'yes';
-			}
-
+			const moduleData = exports.getModuleDataFromHeaders( moduleFile );
```

Comment:
To clarify my comment in https://github.com/WordPress/performance/pull/662#discussion_r1126861733: I suggest you refactor this to something like:
```suggestion
			const moduleHeader = exports.getModuleHeader( moduleFile );

			// Populate slug and focus based on file path.
			const moduleDir = path.dirname( moduleFile );
			const moduleData = {
				slug: path.basename( moduleDir ),
				focus: path.basename( path.dirname( moduleDir ) ),
				...exports.getModuleDataFromHeader( moduleHeader ),
			};
```

Once you have created/updated those functions accordingly, this should work.

Reply:
Thanks @felixarntz for suggestion.